entity-management-definitions/src/main/java/eu/europeana/entitymanagement/web/xml/model/XmlConstants.java

```diff
@@ -86,6 +86,7 @@
   public static final String XML_PHONE = "phone";
   public static final String XML_MBOX = "mbox";
   public static final String XML_HAS_ADDRESS = "hasAddress";
+  public static final String XML_LANGUAGE = "language";
```

Comment:
this is an xml fieldname, therefore it should be placed in parent interface - WebEntityFields

Reply:
The names for all other fields are placed here, I am not sure what you exactly mean.